packages/kbn-router-to-openapispec/src/process_router.ts

```diff
@@ -6,7 +6,7 @@
  * Side Public License, v 1.
  */
 
-import type { Router } from '@kbn/core-http-router-server-internal';
+import { Router } from '@kbn/core-http-router-server-internal';
```

Comment:
NIT: I don't see in the file's short diff why we had to remove the `type` import modifier?